config/badgegenerator.yml

```diff
@@ -1 +1,4 @@
-USER_DATA: "{\"default\":{\"project_public_key\":\"aabbcc\",\"private_key\":\"123456\",\"epochs_max_cu\":1},\"projectId2\":{\"project_public_key\":\"aabbccdd\",\"private_key\":\"12345678\",\"epochs_max_cu\":1}}"
\ No newline at end of file
+USER_DATA: "{\"1\":{\"default\":{\"project_public_key\":\"test111111\",\"private_key\":\"123456\",\"epochs_max_cu\":1},\"projectId2\":{\"project_public_key\":\"test1122\",\"private_key\":\"12345678\",\"epochs_max_cu\":1}},\"2\":{\"default\":{\"project_public_key\":\"test123\",\"private_key\":\"123456\",\"epochs_max_cu\":1}}}"
+DEFAULT_GEOLOCATION: 1
+COUNTRIES_FILE_PATH: "./config/badge/countries.csv"
```

Comment:
add a readme explaining what is expected, after removing these csvs or add a link to another repo with these csv's inside.